WD-Firmware/examples/demo/demo_rtosal.c

```diff
@@ -0,0 +1,470 @@
+/*
+* SPDX-License-Identifier: Apache-2.0
+* Copyright 2019 Western Digital Corporation or its affiliates.
+*
+* Licensed under the Apache License, Version 2.0 (the "License");
+* you may not use this file except in compliance with the License.
+* You may obtain a copy of the License at
+*
+* http:*www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing, software
+* distributed under the License is distributed on an "AS IS" BASIS,
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+* See the License for the specific language governing permissions and
+* limitations under the License.
+*/
+/**
+* @file   demo_rtosal_demo.c
+* @author Nati Rapaport
+* @date   23.10.2019
+* @brief  application of RTOS abstraction layer basic demonstration
+*/
+
+/**
+* include files
+*/
+#include <stdlib.h>
+#include <unistd.h>

```

Comment:
check why do we need these 2 includes

Reply:
done. 
stdlib is now under ifdef D_HI_FIVE1 (it is for 'itoa' used in the application - it is also under ifdef D_HI_FIVE1)
unistd is removed